cmd/file.go

```diff
@@ -248,6 +298,8 @@ func init() {
 	fs.BoolVar(&uploadOpts.ApplyTM, "apply-tm", false, "Enable to automatically apply 100% translation memory matches.")
 	fs.BoolVar(&uploadOpts.HiddenFromContributors, "hidden-from-contributors", false, "Enable to automatically set newly created keys as 'Hidden from contributors'")
 	fs.BoolVar(&uploadOpts.CleanupMode, "cleanup-mode", false, "Enable to delete all keys with all language translations that are not present in the uploaded file. You may want to make a snapshot of the project before importing new file, just in case.")
+	fs.BoolVar(&uploadPooling, "pool", false, "Enable to wait until background file upload finishes with result")
+	fs.DurationVar(&uploadPoolingTimeout, "pool-timeout",30 * time.Second, "Specify custom file upload pooling duration")
```

Comment:
We should set a minimum to 1 second (if it's not yet so) and the default should be 5s.30 - overkill 🙂
+ specify that the duration is is seconds in description

Reply:
It is about maximum pooling time without any result obtained. Updated description